sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -359,6 +359,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val THRIFTSERVER_RESULT_LIMIT =
+    buildConf("spark.sql.thriftServer.retainedResults")
+      .internal()
+      .doc("The number of sql results returned by Thrift Server, and 0 is unlimited.")
```

Comment:
The doc is not clear. Can you add the relation with limit operator into the doc?